### DISCUSSION TITLE
Has anyone tried this with the KICKPI KP1 box?

### DISCUSSION THREAD

**Discussion by qu4rta** [croneter/PlexKodiConnect#2129] [Q&A] [answered]
I'm wondering if someone has tested this with the mentioned android tv box.
Theoretically it's the same as the Homatics R 4k Plus (same specs), but I think it uses different system.
It costs a fraction of the Homatics price, and it would be awesome if it works on it.

**Comment by newoski** [comment, marked as answer]
This isn’t hardware dependent. Unless I’m missing something, if the hardware runs Kodi and you have a Plex server, you should be good to go

**Reply by croneter** [reply]
Yes, agree with @newoski. As long as you're able to run Kodi, PlexKodiConnect should be working just fine
